File: 백준/Bronze/34824. 연대 다음 고대/연대 다음 고대.swift
import Foundation

let N = Int(readLine()!)!
var rankOfYonsei = 0
var rankOfKorea = 0

for i in 0..<N{
    let name = readLine()!
    if name == "yonsei"{
        rankOfYonsei = i
    }else if name == "korea"{
        rankOfKorea = i
    }
}
print(rankOfYonsei > rankOfKorea ? "Yonsei Lost..." : "Yonsei Won!")

File: 백준/Bronze/34824. 연대 다음 고대/연대 다음 고대.cc
#include <iostream>
using namespace std;

int main(int argc, const char * argv[]) {
    int32_t N = 0;
    int32_t rankOfYonsei = 0;
    int32_t rankOfKorea = 0;
    string yonseiStr = "yonsei";
    string koreaStr = "korea";
    
    cin >> N;
    for (int i=0; i<N; i++)
    {
        string name = "";
        cin >> name;
        
        if(name == yonseiStr)
        {
            rankOfYonsei = i;
        }
        else if (name == koreaStr)
        {
            rankOfKorea = i;
        }
    }
    cout << ((rankOfKorea > rankOfYonsei) ? "Yonsei Won!":"Yonsei Lost...");
    return EXIT_SUCCESS;
}